server/src/services/ScheduleServices.js

```diff
@@ -0,0 +1,186 @@
+const FroshGroup = require('../models/FroshGroupModel');
+
+const ScheduleServices = {
+  /**
+   * Get an array of events for a frosh group.
+   * @param {String} froshGroupId the id of the frosh group being accessed
+   * @returns {Object[]}
+   */
+  async getGroupSchedule(froshGroupId) {
+    function formatDate(date) {
+      var hours = date.getHours();
+      var minutes = date.getMinutes();
+      var ampm = hours >= 12 ? 'PM' : 'AM';
+      hours = hours % 12;
+      hours = hours ? hours : 12; // the hour '0' should be '12'
+      minutes = minutes < 10 ? '0' + minutes : minutes;
+      var strTime = hours + ':' + minutes + ' ' + ampm;
+      return strTime;
+    }
+
+    const group = await FroshGroup.findById(froshGroupId);
+    let schedule = {
+      'Monday September 5': [],
```

Comment:
Should this be hard coded?

Reply:
No, this is not what I was picturing when I asked to separate the schedule and the events. I'll refactor this all to fix it